dev/io.openliberty.data.internal_fat/test-applications/DataTestApp/src/test/jakarta/data/web/DataTestServlet.java

```diff
@@ -3438,6 +3438,75 @@ public void testRecordCrudRepositoryMethods() {
         assertEquals(0L, receipts.count());
     }
 
+    /**
+     * Tests all BasicRepository methods with a record as the entity.
```

Comment:
This should be CRUDRepository, right?

Reply:
Yes, I'll get that corrected. Thanks for spotting it.